core/jobs/batch_jobs/exp_migration_jobs.py

```diff
@@ -212,118 +167,20 @@ def _update_exploration(
         ) % (
             feconf.CURRENT_STATE_SCHEMA_VERSION
         )
-        change_dicts = [change.to_dict() for change in exp_changes]
+        models_to_put_values = []
         with datastore_services.get_ndb_context():
-            models_to_put = updated_exp_model.compute_models_to_commit(
+            models_to_put_values = exp_services.update_exploration(
```

Comment:
I think "update" is misleading here, should probably have the naming be around "get models for XYZ"